apps/ui/src/types.ts

```diff
@@ -172,6 +172,7 @@ export type Space = {
   discord: string;
   coingecko?: string;
   terms: string;
+  privacy: 'none' | 'shutter' | 'any';
```

Comment:
This type seems to be used in many places.

Maybe create a new  type `SpacePrivacy`

```ts
export type SpacePrivacy = Privacy | 'any';
```

And use that instead